models/repo_mock.go

```diff
@@ -0,0 +1,25 @@
+package models
+
+type RepoMock struct{}
+
+func (repo *RepoMock) Search(*Query, *[]Cron) error {
+	crons := make([]*Cron, 0)
```

Comment:
Also the method signature is wrong I think i should be `(q *Query, crons *[]Cron )` I think, I will check that